prow/presubmit.yaml

```diff
@@ -76,8 +76,10 @@ kubernetes/kubernetes:
 
 - name: pull-kubernetes-cross
   context: Jenkins Cross Build
-  rerun_command: "@k8s-bot build this"
-  trigger: "@k8s-bot (cross )?build this"
+  always_run: true
+  skip_report: true
+  rerun_command: "@k8s-bot cross test this"
```

Comment:
`@k8s-bot (cross[ -]?)?(build|test) this`

It's really annoying to try a build command that just doesn't work because it's the wrong regex.